crates/autopilot/src/database/competition.rs

```diff
@@ -34,6 +37,26 @@ pub struct Competition {
     pub competition_table: SolverCompetitionDB,
 }
 
+/// Contains the reference score per solver, defined as the difference between
+/// the total score for the auction and the combined score of all winning
+/// solutions submitted by that solver for the same auction.
+#[derive(Clone, Default, Debug)]
+pub struct ReferenceScores(HashMap<H160, U256>);
+
+impl ReferenceScores {
+    pub fn iter(&self) -> Iter<'_, H160, U256> {
```

Comment:
nit: This return type is needlessly specific. It's a good rule of thumb to return `impl Iterator` because that way you don't have to change anything if you for example apply a `.filter()` to the current iterator.
```suggestion
    pub fn iter(&self) -> impl Iterator<Item = (&H160, &U256)> {
```